### ISSUE TITLE
Kafka Broker not exposing RMI JMX statistics on correct port 

### ISSUE THREAD

**Issue by Alfapalooza** [bitnami/charts#1291]
**Which chart**:
bitnami/kafka:latest

**Description**
When using the `values-production.yaml` configuration the `jmx-exporter` container that's run alongside the `kafka-broker` container in the `kafka` pod is reporting an exception:
```
SEVERE: JMX scrape failed: java.io.IOException: Failed to retrieve RMIServer stub: javax.naming.ServiceUnavailableException [Root exception is java.rmi.ConnectException: Connection refused to host: 127.0.0.1;
```
The exporter is configured to fetch RMI JMX statistics from the broker over port "5555" by default according to this chart config `metrics.jmx.jmxPort`, however the `kafka-broker` is not configured to report it's RMI JMX statistics over that port. Narrowed it down to a missing `kafka-broker` config:
```
- name: JMX_PORT
  value: "5555"
```

**Describe the results you expected:**
Expecting to see a `kafka-broker` config entry for `JMX_PORT` that corresponds to `metrics.jmx.jmxPort` helm chart config.

**Comment by marcosbc** [comment]
Hi, just wanted to let you know we are looking into this and will get back to you when we have any update.

**Comment by marcosbc** [comment]
Hi, we have looked into this. The container works fine for us, in fact the `JMX_PORT` envvar was already added time ago.

We did see the error in one of the pods, but it was happening because Kafka was not yet ready at that point. For instance when we deleted the pod and it got recreated, the warning was not present anymore. See [templates/statefulset.yaml#L117](https://github.com/bitnami/charts/blob/master/bitnami/kafka/templates/statefulset.yaml#L117).

Let us know if we're missing anything.
